files/views.py

```
@@ -496,7 +513,211 @@ def edit_media(request):
+                        except Exception as close_error:
+                            logger.warning(
+                                f"Failed to close temporary file handle for media {media.friendly_token}: {close_error}"
+                            )
+
+                # Clean up temporary upload files after successful save
+                try:
+                    if temp_file_path and os.path.exists(temp_file_path):
+                        rm_file(temp_file_path)
+                    if upload_file_path and os.path.exists(upload_file_path):
+                        shutil.rmtree(upload_file_path)
+                except Exception as e:
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)